broadway_api/daos/base.py

```diff
@@ -1,15 +1,44 @@
+import bson
+import logging
+
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+logger = logging.getLogger("dao")
+
+
+def validate_obj_size(func):
```

Comment:
Nit: if this is a decorator, I'd argue that it should be in `broadway_api/daos/decorators.py`. It's not really a base dao

Reply:
Fair point

Reply:
Note that we have a `broadway_api/decorators` module, but since this one is internal to the DAOs, I think having a separate `broadway_api/daos/decorators.py` makes sense.

Having the "public" decorators in `broadway_api/decorators` still feels right.

Reply:
Yep noticed that but since this decorator is heavily dao dependent, having it is `broadway_api/daos/decorators.py` made the most sense.